opentech/apply/funds/models/mixins.py

```diff
@@ -97,7 +97,7 @@ def data(self, id):
         definitive_id = self.get_definitive_id(id)
         try:
             return self.raw_data[definitive_id]
-        except KeyError as e:
+        except KeyError:
```

Comment:
Error F841 from Flake8, better to fix this than to ignore since only one instance of it.